docker-compose.yml

```diff
@@ -3,6 +3,7 @@ services:
   zabbix-proxy:
     restart: unless-stopped
     image: ${CONTAINER_IMAGE}:${CONTAINER_VERSION}
+    user: 1343
```

Comment:
The UID is installation specific so it's not good to hardcode it. Instead there should be default (e.g. 1000) that can be overridden when creating the proxy.